src/tests/groupby/groupby-kernels-test.cu

```diff
@@ -383,50 +383,7 @@ typedef ::testing::Types<
 
   TYPED_TEST_CASE(GroupByTest, Implementations);
 
-TYPED_TEST(GroupByTest, DISABLED_AggregationTestHost)
```

Comment:
Hmm I don't think it OK to remove these tests. Can you please add them back and get them working? Thanks!

Reply:
These tests test the host-side implementation of the `concurrent_unordered_multimap` and `concurrent_unordered_map` classes insert functions.  

I'm in the process of updating these classes to no longer have host-side implementations of these functions as they require the class to use managed memory, which complicates the custom memory manager that is being developed. 

Furthermore, there is no good reason to have host-side implementations of these functions and maintaining them just adds unnecessary work and code bloat.

Reply:
That makes sense, but could you add device side tests to replace them?

Reply:
See `/src/tests/hash_map/map-test.cu`.

However, this brings up a good question related to unit testing. For example, should unit tests just test the top-level libgdf APIs in `functions.h`? Or should there also be unit testing for the classes/functions/kernels that are used to implement the top level API? 

In this case, we're talking about tests for the lower-level implementation classes, which are turning out to be a lot of work to maintain as implementations are updated.

I'll pose the question in Slack.

Reply:
Thanks! Yeah it would be great to get some clarity on what precisely needs to be tested.

Reply:
We are trying, with lukewarm results, to create unit tests for most functions we make. So not just the top level apis but the lower level functions that get leveraged as well. If we only test top level apis those are not unit tests. Those are integration tests. And those are great but unit tests really should test only what happens in the function and mock up whatever happens outside of it.